en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -57,11 +57,15 @@ During deployment, Choreo performs a Trivy scan to detect critical vulnerabiliti
 
 ## Environment variables
 
-You can use environment variables in WSO2 Micro Integrator to improve the configuration management, security, portability, and manageability of the Micro Integrator installation. By using environment variables, organizations can simplify the management and maintenance of their micro installations and ensure that they can be quickly and easily updated when required.
+It is recommended to use environment variables when developing integration artifacts with WSO2 Integration Studio to improve configuration management, security, portability, and manageability. By using environment variables, organizations can simplify the management and maintenance of their integrations and ensure that they can be quickly and easily updated when required.
```

Comment:
Are you trying to say that we recommend that they use environment variables? or is it as an alternative to another option?

Reply:
We recommend that they use environment. It is the better option here.